tidy3d/components/geometry/polyslab.py

```diff
@@ -1316,6 +1317,13 @@ def _heal_polygon(vertices: np.ndarray) -> np.ndarray:
         shapely_poly = PolySlab.make_shapely_polygon(vertices)
         if shapely_poly.is_valid:
             return vertices
+        elif isbox(vertices):
+            raise NotImplementedError(
+                "It looks like the dilation causes damage to the polygon. "
```

Comment:
Let's just say "The dilation caused damage to the polygon. ", same with the error message in JaxPolySlab ('it looks like' sounds slightly strange in an error message to me).